src/components/rockets/oneRocket.js

```diff
@@ -0,0 +1,71 @@
+import { useDispatch } from 'react-redux';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { bookingRocket, cancelRocket } from '../../redux/rockets/rockets';
+import './oneRocket.css';
+
+const OneRocket = ({ rockets }) => {
+  const {
+
+    id,
+    rocketName,
+    description,
+    flickrImages,
+    reserved,
+  } = rockets;
+
+  const dispatch = useDispatch();
+
+  const booking = (id) => {
+    dispatch(bookingRocket(id));
+  };
+
+  const canceling = (id) => {
+    dispatch(cancelRocket(id));
+  };
+
+  return (
+    <div className="rocket" id={id}>
+      <img src={flickrImages} alt="" />
+      <div className="column">
+        <h2 className="rocket-name">{rocketName}</h2>
+        <div className="badge">
+
+          { reserved
+            ? <button type="button" className="reserved">reserved</button>
+            : ''}
+
+          <p className="rocket-description">{description}</p>
+        </div>
+        { reserved
+          ? <button onClick={() => canceling(id)} className="cancel-button" type="button">Cancel Rocket</button>
+          : <button type="button" onClick={() => booking(id)} className="booking-button">Reserve Rocket</button>}
+      </div>
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/156244653-5599d59b-03ea-4689-aeb5-e6565b48134c.png)

Therefore:

- [ ] Please, kindly use the React conditional rendering syntax, instead of using ternary operators.

## Hints

You could build if-else statements using this syntax by using the `!` (NOT) operator like this: `{!rocket.reserved && (what to do if the previous thing is true)}` and still meet the requirement 💪